presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableScanNode.java

```diff
@@ -77,6 +79,14 @@ public TableScanNode(
         requireNonNull(tableLayout, "tableLayout is null");
         requireNonNull(currentConstraint, "currentConstraint is null");
         checkArgument(currentConstraint.isAll() || tableLayout.isPresent(), "currentConstraint present without layout");
+        currentConstraint.getDomains()
+                .map(Map::keySet)
```

Comment:
nit: move `.keySet()` inside `.ifPresent` and put `.ifPresent` on the same line as `currentConstraint.getDomains()`